### PR TITLE
Removed a ton of ad code that we no longer need.

### PR DESCRIPTION
**Pull request by hworld**

### CODE REVIEW

File: components/ad/placement/placement.ts [hunk 1]
```diff
@@ -3,8 +3,7 @@ import { AppAdWidget } from 'game-jolt-frontend-lib/components/ad/widget/widget'
 import Vue from 'vue';
 import { Component, Prop } from 'vue-property-decorator';
 import { Screen } from '../../screen/screen-service';
-import { Ads } from '../ads.service';
-import { AdSlotPos, AdSlotPosValidator } from '../slot';
+import { Ads, AdSlotPos, AdSlotPosValidator } from '../ads.service';
```
Comment (nilllzz) [inline components/ad/placement/placement.ts:6]: Why is this imported and not used?

File: components/ad/playwire/playwire.html [hunk 2]
```diff
@@ -1,7 +1,9 @@
-<div
-	v-if="slotId"
-	:key="slotId"
-	>
-	<div :data-pw-desk="placement" />
-	<div :data-pw-mobi="placement" />
+<div>
```
Comment (YLivay) [inline components/ad/playwire/playwire.html:1]: why?

File: components/ad/widget/widget.ts [hunk 3]
```diff
@@ -1,14 +1,20 @@
 import View from '!view!./widget.html?style=./widget.styl';
-import { AppAd } from 'game-jolt-frontend-lib/components/ad/ad';
-import { Ads, hasPlaywire } from 'game-jolt-frontend-lib/components/ad/ads.service';
+import {
+	Ads,
+	AdSlotPos,
+	AdSlotPosValidator,
```
Comment (YLivay) [inline components/ad/widget/widget.ts:5]: Remove unused imports

File: components/ad/playwire/playwire.ts [hunk 4]
```diff
@@ -65,21 +103,17 @@ export class AppAdPlaywire extends Vue {
 
 	beforeDestroy() {
 		Playwire.removeAd(this);
-		// removeClickTracker(this.$el);
+		removeClickTracker(this.$el);
 	}
 
 	display() {
```
Comment (YLivay) [inline components/ad/playwire/playwire.ts:109]: what is this function? what uses it?

**Pull request merged**